misc/openapi-ballerina/modules/ballerina-to-openapi-generator/src/main/java/org/ballerinalang/ballerina/openapi/convertor/service/OpenApiResourceMapper.java

```diff
@@ -64,7 +64,7 @@
 /**
  * This class will do resource mapping from ballerina to swagger.
  */
-public class SwaggerResourceMapper {
+public class OpenApiResourceMapper {
     private final String httpAlias;
     private final String swaggerAlias;
     private final Swagger swaggerDefinition;
```

Comment:
can we change the "swaggerDefinition" and "swaggerAlias" ?

Reply:
Resolved with 81d9190